src/platform_impl/macos/appkit/window.rs

```diff
@@ -267,6 +267,8 @@ use window_level_key::*;
 #[derive(Clone, Copy, Debug, PartialEq, Eq, Hash)]
 #[repr(isize)]
 pub enum NSWindowLevel {
+    #[doc(alias = "BelowNormalWindowLevel")]
+    BelowNormal = (kCGBaseWindowLevelKey - 1) as _,
```

Comment:
I looked into how window levels actually work, and it turns out that what we're doing currently is wrong, see https://github.com/rust-windowing/winit/pull/2545.

I'm fine with the implementation of `BelowNormal` though, but it'll have to be updated to use `kCGNormalWindowLevel - 1` after that PR lands.

Reply:
I've merged that PR now, so you should be able to rebase now

Reply:
(Perhaps name the new constant something like `BELOW_NORMAL` to differentiate it from the other ones that are meant to closely mimic the ones available in the system's `NSWindowLevel`)

Reply:
Sure